src/lib/paths.ts

```diff
@@ -65,12 +66,21 @@ export const ensureDir = async (dirPath: string): Promise<string> => {
 
 // User .fluence paths:
 
-export const ensureUserFluenceDir = async (
+export const ensureUserFluenceDir = (
   commandObj: CommandObj
-): Promise<string> =>
-  commandObj.config.windows
-    ? ensureDir(commandObj.config.configDir)
-    : ensureDir(path.join(os.homedir(), FLUENCE_DIR_NAME));
+): Promise<string> => {
+  const globalFluenceDirPathFromEnv = process.env[FLUENCE_USER_DIR];
+
+  if (typeof globalFluenceDirPathFromEnv === "string") {
+    return ensureDir(globalFluenceDirPathFromEnv);
+  }
+
+  if (commandObj.config.windows) {
+    return ensureDir(commandObj.config.configDir);
```

Comment:
Is `config.configDir` an option to CLI, like `--config-dir`? Or where does it come from?

Reply:
If I used default configDir for unix as well - maybe it would be easier. From oclif docs

configDir - CLI config directory
Unix: ~/.config/mycli
Windows: %LOCALAPPDATA%\mycli
Can be overridden with XDG_CONFIG_HOME

we can discuss this

Reply:
`XDG_CONFIG_HOME` doesn't sounds like anything good to me :D Sounds scary and unrelated to `fluence cli`.

> There is a tool xdg, which manages application integration with the different GUI Desktops in the Linux world. 

https://wiki.archlinux.org/title/XDG_user_directories

I don't think we want to use these, unless someone has a really good understanding of these conventions and how they interoperate with POSIX/X window/WIndows etc.

I've personally no idea, to me it's just some scary shit from 00s.